py.requirements/testing.txt

```diff
@@ -20,5 +20,9 @@ PyHamcrest <  2.0;   python_version <  '3.0'
 # HINT: path.py => path (python-install-package was renamed for python3)
 path.py >=11.5.0,<13.0; python_version <  '3.5'
 path >= 13.1.0;         python_version >= '3.5'
+# NOTE: toml-0.10.2 is required for python<3 (tomli not available)
+tomli>=1.1.0; python_version >=  '3.0' and python_full_version < '3.11.0a7'
+toml>=0.10.2; python_version <  '3.0'
+
```

Comment:
It might be possible to refer to setup.py's `[toml]` extras like this:

```suggestion
.[toml]
```

Do you want to try?